src/image_processing/pgm.py

```diff
@@ -0,0 +1,72 @@
+import csv
+import sys
+import argparse
+import os
+import pathlib
+
+from plotting_utils.plotting_helper import file_arg, path_arg, int_arg_positive_nonzero, check_aedat_csv_format
+
+
+def get_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "aedat_csv_file", help="CSV containing AEDAT data to be plotted (On,Off,Both,PGM_String)", type=file_arg
```

Comment:
```suggestion
        "aedat_csv_file", help="CSV containing AEDAT data to be plotted. Must contain a PGM_String column", type=file_arg
```
We should change the help message so that it is clear that we only need a PGM_String column